pages/insights/index.njk

```diff
@@ -20,10 +20,14 @@ hero:
 
     <div class="grid-container padding-top-205">
       <div class="grid-row grid-gap">
-      {% for insight in collections.insights | sort(attribute='sortorder') %}
+      {% for insight in collections.insights | sort(attribute='data.sortorder') %}
         <div class="tablet:grid-col-4 margin-y-1 padding-bottom-6">
           <div class="flex-column">
-          <a class="usa-link text-no-underline text-black" href="{{ insight.url | url }}" aria-label="link to {{ insight.data.title}}">
+             {% if insight.data.externallink %}
+               <a class="usa-link text-no-underline" href="{{ insight.data.externallink | url }}">
```

Comment:
@weiwang-gsa Do we need to also include the `aria-label` here too for accessibility?